tests/docker-compose.yml

```diff
@@ -9,21 +9,11 @@ postgresql:
     ports:
         - "55432:5432"
 elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.8
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.7.0
```

Comment:
Or is it `6.6.0`?

Reply:
6.7.0 should be used everywhere, but it doesn't install on travis, so I've used 6.6.0 there until i figure out what is wrong.